code/modules/admin/admin_verbs.dm

```diff
@@ -68,6 +68,7 @@ GLOBAL_PROTECT(admin_verbs_admin)
 	/client/proc/cmd_admin_world_narrate, /*sends text to all players with no padding*/
 	/client/proc/cmd_change_command_name,
 	/client/proc/create_mob_worm,
+	/client/proc/spawn_as_mmi,
```

Comment:
this list needs to be alphabetized, there's extensions for vscode that will automatically do this for you.